datahub-web-react/src/app/entityV2/summary/useShowAssetSummaryPage.ts

```diff
@@ -0,0 +1,9 @@
+import { useAppConfig } from '@app/useAppConfig';
+
+export function useShowAssetSummaryPage() {
+    const appConfig = useAppConfig();
+    if (appConfig?.loaded) {
+        return appConfig.config.featureFlags.assetSummaryPageV1;
+    }
+    return false;
+}
```

Comment:
we're going to want this to work with localStorage like we do with other feature flags so that when we refresh the page we don't see things flashing

Reply:
i have a ticket for us to handle this later after more important things are done https://linear.app/acryl-data/issue/CH-697/add-feature-flag-to-localstorage